scripts/aem.js

```diff
@@ -544,4 +559,52 @@ export class Brick extends HTMLElement {
 
     this.observer.observe(this.shadowRoot, { childList: true, subtree: true });
   }
+
+  get config() {
```

Comment:
Naming freak again...IIUC this is the _brick_ config? I'd call the getter `brickConfig` then.

Reply:
Brick is the class already, and config is a getter of Brick. Why add a `brick` prefix to the method?

Reply:
> ...Brick is the class already...

Ah sorry I missed that, all good then. Sorry for the noise!